### PR TITLE
List component

### PR DESCRIPTION
**Pull request by prath-dev**
List Component Added .
Live Preview : http://127.0.0.1:5501/index.html#list

### CODE REVIEW

**Comment by MonikaJhingan** [comment]
Code is indented well just keep in mind regarding the extra blank spaces in the html file.

**Comment by prath-dev** [comment]
> Code is indented well just keep in mind regarding the extra blank spaces in the html file.

okkk thanks noted.

File: components/List/list.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+.orderlist-container {
+    margin-bottom: 5rem;
+    height: 18rem;
+    width: 48rem;
+    border: 2px solid black;
+    display: flex;
+    flex-direction: column;
+    /*margin-bottom: 1rem ;*/
```
Comment (muskan-gupta-0803) [inline components/List/list.css:28]: You can remove the commented code

**Review by MonikaJhingan — APPROVED** [review]
Code looks good to be merged just add meta tag for author and description in later projects.

File: components/List/list.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    margin-right: 2rem;
+}
+
+.list-style-circle {
+    margin-right: 2rem;
+    list-style-type: circle;
+}
+
+.list-style-square {
+    margin-right: 2rem;
```
Comment (Sharmaryan) [inline components/List/list.css:42]: you have given same margin, what you can do you can make a class with any name and give that margin right : 2rem and then add that class to every element where that margin is required, we have to follow DRY principle(do not repeat yourself).

File: components/List/list.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="list.css" rel="stylesheet" type="text/css" />
+    <title>Document</title>
```
Comment (Sharmaryan) [inline components/List/list.html:9]: fix : title

**Review by Sharmaryan — APPROVED** [review]
suggested some changes, you can make these changes otherwise everything is fine

**Review by Megha-Joshi — APPROVED** [review]
Looks good to me. You can merge it.

File: components/List/list.css (hunk 2 above)
Comment (prath-dev) [reply components/List/list.css:42]: okkk thanks for your suggestion

Comment (prath-dev) [reply components/List/list.css:42]: okkk nice suggestion

**Pull request merged**